docs/deploy-cluster-on-docker.md

```diff
@@ -0,0 +1,226 @@
+                    "IPAddress": "172.17.0.3",
+$ docker inspect 591e2f6f48e2 | grep IPAddress
+            "SecondaryIPAddresses": null,
+            "IPAddress": "172.17.0.2",
+                    "IPAddress": "172.17.0.2",
+```
+
+Therefore, this article will deploy Nebula cluster on three hosts as follows:
+
+```
+172.17.0.2 # cluster-2: metad/storaged/graphd
+172.17.0.3 # cluster-3: metad/storaged/graphd
+172.17.0.4 # cluster-4: metad/storaged/graphd
+```
+
+## Configuration
+All the configuration files of Nebula are located in `/usr/local/nebula/etc`, and three default configuration files are provided there. Edit them separately:
+
+First configuration file: **nebula-metad.conf**
+
+Metad ensures high availability through the raft protocol. So you need to configure the ip and port deployed for each metad service. You need to configure `meta_server_addrs` and `local_ip` and leave other parameters with the default values. Consider cluster-2 configuration as example:
+
+```
+# Peers
+--meta_server_addrs=172.17.0.2:45500,172.17.0.3:45500,172.17.0.4:45500
+# Local ip
+--local_ip=172.17.0.2
+# Meta daemon listening port
+--port=45500
+```
+
+Second configuration file: **Nebula-graphd.conf**
+
+Graphd needs to obtain the Schema data from metad when running, so metad's IP address and port `meta_server_addrs` must be specified in configuration, other parameters can use the default values. Consider cluster-2 configuration as example:
+
+```
+# Meta Server Address
+--meta_server_addrs=172.17.0.2:45500,172.17.0.3:45500,172.17.0.4:45500
+```
+
+Third configuration file: **Nebula-storaged.conf**
+
+Storaged also uses the raft protocol to ensure high availability and communicates with metad during data migration. Therefore, you need to configure the address and port `meta_server_addrs` of metad and the `local_ip`, which can be obtained by its peers through metad. Consider cluster-2 configuration as example:
+
+```
+# Meta server address
+--meta_server_addrs=172.17.0.2:45500,172.17.0.3:45500,172.17.0.4:45500
+# Local ip
+--local_ip=172.17.0.2
+# Storage daemon listening port
+--port=44500
+```
+
+Repeat the above steps to configure cluster-3 and cluster-4. A total of 9 files must be configured.
+
+## Run Clusters
+
+After login, you're in the `root` directory and you should switch to the nebula directory
+
+```bash
+$ cd /usr/local/nebula/
+```
+
+Restart the service after the cluster configuration is complete:
+
+```bash
+$ /usr/local/nebula/scripts/nebula.service stop all
+[INFO] Stopping nebula-metad...
+[INFO] Done
+[INFO] Stopping nebula-graphd...
+[INFO] Done
+[INFO] Stopping nebula-storaged...
+[INFO] Done
+
+$ /usr/local/nebula/scripts/nebula.service start all
+[INFO] Starting nebula-metad...
+[INFO] Done
+[INFO] Starting nebula-graphd...
+[INFO] Done
+[INFO] Starting nebula-storaged...
+[INFO] Done
+```
+
+Repeat the above command to restart cluster-3 and cluster-4.
+
+## Test Clusters
+Log on to one of the clusters and run the following command:
```

Comment:
add `Use the console to access the cluster.`
Log on to one of the clusters and run the following command